package.json

```diff
@@ -16,8 +16,8 @@
     "build": "spago -x bin/spago.dhall build",
     "test": "spago -x test/spago.dhall test",
     "generate-default-operators": "spago -x script/spago.dhall run -m GenerateDefaultOperatorsModule",
-    "format-self": "npm run build && node ./bin/index.js format-in-place \"src/**/*.purs\" \"test/*.purs\" \"bin/**/*.purs\" \"script/**/*.purs\"",
-    "check-self": "npm run build && node ./bin/index.js check \"src/**/*.purs\" \"test/*.purs\" \"bin/**/*.purs\" \"script/**/*.purs\"",
+    "format-self": "npm run build && node ./bin/index.js format-in-place src test bin script",
```

Comment:
Test should still be `'test/*.purs'` because you don't want it to be recursive (snapshot files).

Reply:
I used `test/Test` instead to match anything in that directory -- that way if we add directories in the future we're OK.

Reply:
But our files are not in `test/Test`.

Reply:
🎵 let's pretend this never happened 🎵